recipes/python3/__init__.py

```diff
@@ -32,12 +32,39 @@ def prebuild_arch(self, arch):
         self.append_file("ModulesSetup.mobile", "Modules/Setup.local")
         self.apply_patch("xcompile.patch")
         self.set_marker("patched")
+    
+    def postbuild_arch(self, arch):
```

Comment:
No, this should not be needed. Why this and not all the others modules declares in Modules?
Something is wrong here. if _ssl, _struct, array, _socket... are working, _sqlite3 should work the same, as it is declared the same way. Could it be the inclusion of `*static*` ?

Reply:
Not related to `*static*`. I also supposed that wasn't needed, but I've found out that for some reason, `python` itself builds this module as a `lib-dynload` also if it is supposed to "merge" it statically inside `libpython3`.